dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/handler/NettyClientHandler.java

```diff
@@ -187,9 +187,7 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
             heartBeat.setBody(heartBeatData);
             ctx.channel().writeAndFlush(heartBeat)
                     .addListener(ChannelFutureListener.CLOSE_ON_FAILURE);
-            if (logger.isDebugEnabled()) {
-                logger.debug("Client send heart beat to: {}", ChannelUtils.getRemoteAddress(ctx.channel()));
-            }
+            logger.debug("Client send heart beat to: {}", ChannelUtils.getRemoteAddress(ctx.channel()));
```

Comment:
In fact, I don't agree with remove `logger.isDebugEnabled` here, use `logger.debug` will need to calculate `ChannelUtils.getRemoteAddress(ctx.channel())` in each log, but in most of times we don't open debug, we don't need to calculate this.